src/tv2-common/actions/executeAction.ts

```diff
@@ -192,6 +191,11 @@ export interface ActionExecutionSettings<
 		alphaAtEnd: number
 	) => WithTimeline<VTContent>
 	pilotGraphicSettings: PilotGeneratorSettings
+	serverActionSettings: ServerActionSettings
+}
+
+interface ServerActionSettings {
+	defaultTriggerMode: ServerSelectMode
```

Comment:
The type is `ServerSelectMode` so it would make more sense to call the variable `defaultServerSelectMode`.

But I think there is a general mismatch with the naming for `ServerSelectMode`. `Select Mode` refers to **how** the Server can be selected, but the values of ServerSelectMode, `RESUME` and `RESET`, refers to **what** happens **after** the Server has been selected i.e. how it should be played.

So an even more accurate naming would something along the lines of:
```
interface ServerActionSettings {
  defaultPlaybackBehaviour: PlaybackBehaviour
}
```

I can see the naming `triggerMode` has been used throughout for this. Not saying to refactor it now, but I would highly suggest we refactor it as this specific enum has nothing to do with how it's been "triggered".